defaults/main.yml

```diff
@@ -1,4 +1,6 @@
 ---
+erlang_repo: False
```

Comment:
i think this should be `true` by default, and overridden in edge cases like yours to be false. :)

Reply:
I am wondering about this. I somewhat was thinking that it would be actually great to use whatever the distribution is providing by default, because the erlang repository is something that is out of the ordinary with regards to the repo setup.

@jasonroyle , do you have any thoughts on this?